shapely/geometry/polygon.py

```diff
@@ -388,22 +387,6 @@ def asPolygon(shell, holes=None):
     return PolygonAdapter(shell, holes)
 
 
-def orient(polygon, sign=1.0):
-    s = float(sign)
-    rings = []
-    ring = polygon.exterior
-    if signed_area(ring)/s >= 0.0:
-        rings.append(ring)
-    else:
-        rings.append(list(ring.coords)[::-1])
-    for ring in polygon.interiors:
-        if signed_area(ring)/s <= 0.0:
-            rings.append(ring)
-        else:
-            rings.append(list(ring.coords)[::-1])
-    return Polygon(rings[0], rings[1:])
-
-
```

Comment:
If we `from shapely.ops import orient` in polygon.py, we won't break any code that depends on importing it from here.

Reply:
Actually adding that import causes circular dependency issue. I've implemented your suggestion in a little different way.